mlflow/tensorflow/__init__.py

```diff
@@ -407,7 +428,13 @@ def save_model(
         # To maintain prior behavior, when the format is HDF5, we save
         # with the h5 file extension. Otherwise, model_path is a directory
         # where the saved_model.pb will be stored (for SavedModel format)
-        file_extension = ".h5" if save_format == "h5" else ""
+        # For keras 3.0, it only supports saving model in .h5 or .keras format
```

Comment:
does this change start from 2.15? keras 3.0 is not a valid version number now.

Reply:
It starts from 2.16.0 dev version, will update the comment to be tensorflow version